kiosk/settings/base.py

```diff
@@ -70,6 +71,7 @@
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'corsheaders.middleware.CorsMiddleware',
+    'whitenoise.middleware.WhiteNoiseMiddleware',
```

Comment:
added whitenoise package as a middleware for deployment.
Without this, the browser shows a white page.